vite.config.js

```diff
@@ -8,7 +8,8 @@ config.LCH = config.COLOR_FN !== '"oklch"'
 export default defineConfig({
   define: config,
   build: {
-    assetsDir: '.'
+    assetsDir: '.',
+    assetsInlineLimit: 0
```

Comment:
Strange, I can’t find any `data:` string in [the current source code](view-source:https://oklch.evilmartians.io/)

```html
<link rel="apple-touch-icon" href="[/apple-touch-icon.png](view-source:https://oklch.evilmartians.io/apple-touch-icon.png)"/>
```

Do we need these changes?

Reply:
Now the image is in the `public` folder. Currently no cache buster is applied to the file, so there is no `data:` in the current source code.

I'll check back in a few hours and see how it works without changing the parameter `assetsInlineLimit` :)

Reply:
Yes you are right!
We don't need to disable this setting because apple-touch-icon more than [default limit](https://vitejs.dev/config/build-options.html#build-assetsinlinelimit) `4 KB` — `17 KB` and `19 KB`.

I removed this rule.